paper/paper.md

```
@@ -0,0 +1,136 @@
+
+# Workflow at a Glance
+
+![The Directed Acyclic Graph (DAG) of the complete long-read RNA-Seq workflow. The graph was generated by the Snakemake command: 'snakemake --rulegraph | dot -Tsvg > rulegraph.svg'.](rulegraph.svg)
+
+TODO: Annotate the workflow with boxes per feature set. Requires latests updates and cleanups to be incorporated
+
+## Example Dataset
+
+A complete example run using six cDNA Nanopore sequencing datasets from human H9 and HCT 116 cell lines is provided below. The data originate from the SG-NEx project [@Chen2025-yk] and were accesed on 31 March 2025 from [registry.opendata.aws/sg-nex-data](registry.opendata.aws/sg-nex-data). This example illustrates the workflows capabilities in isoform analysis and functional annotation.
+
+[]() # Add report HTML
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)